index.js

```diff
@@ -110,3 +44,9 @@ aContact.addEventListener('click', () => {
   document.querySelector('#form-section').classList.remove('active');
   document.querySelector('#contact-section').classList.add('active');
 });
+
+function init() {
+  getTime();
+  Store.getBooks().forEach((book) => UI.getAllBooks(book));
```

Comment:
- [ ] it would be better if you use the es6 `arrow function` instead of the `function keyword` in order to practice to work with both